.github/workflows/cleanup-staging.yml

```diff
@@ -0,0 +1,33 @@
+name: Cleanup staging URLs
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Setup GCloud
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - name: Delete preview on GCS
+        run: gsutil -m rm -r gs://staging.nodejs.dev/${{ github.event.client_payload.pull_request.number }}/
+
+      - name: Comment on PR
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Deleted preview at: `https://staging.nodejs.dev/${{ github.event.client_payload.pull_request.number }}/`'
```

Comment:
where did you see that it should be `client_payload.pull_request.number`???

I was using `github.event.issue.number`

Reply:
This is an event on the `pull_request` rather than the `issue` so the payload/context is different from the old job.
Pretty sure I got it from https://blog.stephencleary.com/2020/04/github-actions-pull-request-staging-environments-part-3-implementing-deploy-and-teardown.html